docs/content/en/docs/development.md

```diff
@@ -34,7 +34,7 @@ For the best experience, we recommend following these steps:
   Just as with bazel, you will want to move this somewhere on your PATH, e.g.:
 
   ```bash
-  mkdir ~/bin
+  mkdir -p ~/bin
```

Comment:
Does this change imply there are systems with no home directory, but for which `~` has a meaningful resolution in the shell?

Reply:
Probably not xD -  I'm just using it because it won't throw a "cannot create directory <...>: File exists" (we tell people to mkdir ~/bin both for bazel in getting_started.md and for buildifier in development.md)